content/en/docs/Tasks/packaging-an-operator.md

```diff
@@ -233,6 +233,41 @@ TODO: example status
 
 `nativeAPIs` is an optional field, but the more information you give OLM about the context in which your operator should be run, the more informed decisions OLM can make.
 
+## Packaging Additional Objects Alongside an Operator
+
+Operators can include additional objects alongisde their `CSV` in the `/manifests` directory. These objects should be YAML files and valid kuberenetes objects. The following objects are supported as of OLM 0.16.0:
```

Comment:
Just leaving this comment here so that we remember:  we'll have to refactor this once we have versioned document.